iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1387,21 +1387,21 @@
 
 "type.amenity.recycling.centre" = "Recycling Center";
 
-"type.amenity.recycling" = "Contenedor de reciclaje";
+"type.amenity.recycling" = "Recycling Container";
```

Comment:
@pastk can we patch twine to use base (es) localization here instead of en? We had an issue for it somewhere, right? It is needed for iOS only at the moment, as Android works properly by using the base language values.

Reply:
https://github.com/organicmaps/organicmaps/issues/1757
a proper fix should be in iOS though, not in Twine

Reply:
I created a patch (https://github.com/organicmaps/twine/pull/3) to add regional language fallback to generic language

Reply:
Great! Does it work with our strings?

Reply:
> It can be done either in iOS code or at the generation time.

Yes, but if done in iOS
- we'll get rid of duplication in localization files and, more importantly, associated diff noise
- we won't have to redo it in case we migrate away from Twine

and if done in Twine
- there will be a tad less iOS run-time code
But its negligible.

Reply:
If it's already done and works, let's use it )